integration-tests/integration/common/common.spec.js

```diff
@@ -541,6 +541,28 @@ And('I should see the {string} query have the value {string}', ($string, $value)
   })
 })
 
+// common pac filter panel specs
+
+When('I click the {string} filter button', $string => {
+  cy.get('.pac-filter-button').contains($string).click()
+})
+
+When('I select the {string} filter', $string => {
+  cy.get(`input[data-test="${$string}"]`).invoke('attr', 'checked', true)
+})
```

Comment:
Do we use data-test or data-cy? I think I've used cy and shouldn't!

Reply:
we have both in the code base. what is the preferred one?

Reply:
I don't actually mind at all